src/components/EventUserTable.js

```diff
@@ -132,6 +51,38 @@ export class EventUserTable extends Component {
   }
 
   render() {
+    /**
+     * Helper function to determine whether to display action for check-in or undo check-in
+     * @param {*} rowData data about the current row
+    */
+    const changeRegistration = (event, rowData) => {
+      switch (event.target.value) {
+        case REGISTRATION_STATUS.REGISTERED:
+          if (window.confirm("Do you want to register " + rowData.fname + " " + rowData.lname + "?\nThis will send an email to the user.")) {
```

Comment:
Consider https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
wait this is hella cool, i added it but it looks kinda weird for the second line since i need to remove all the indenting before it in order for it to be aligned with the first line